packages/qdrant-loader/src/qdrant_loader/webhooks/handlers.py

```
@@ -0,0 +1,73 @@
+    except Exception:
+        return "<non-serializable payload>"
+    if len(summary) > 1000:
+        return summary[:1000] + "..."
+    return summary
+
+
+async def process_webhook_event(
+    project_id: str | None,
+    source_type: str,
+    source: str,
+    payload: Any,
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `payload`

(ANN401)